heliopy/data/util.py

```diff
@@ -13,6 +13,7 @@
 import astropy.units as u
 import sunpy.timeseries as ts
 import warnings
+from collections import OrderedDict
```

Comment:
The problem is that the documentation is trying to document `OrderedDict`. You need to do something like

```python
import collections as coll
.
.
.

coll.OrderedDict(...)
```